pkg/corerp/renderers/gateway/render.go

```diff
@@ -328,6 +329,32 @@ func MakeRoutesHTTPProxies(ctx context.Context, options renderers.RenderOptions,
 			}
 		}
 
+		var timeoutPolicy *contourv1.TimeoutPolicy
+		if route.TimeoutPolicy != nil {
+			// Parse the timeout policy request durations and compare them
+			requestDuration, err := time.ParseDuration(route.TimeoutPolicy.Request)
+			if err != nil {
+				return []rpv1.OutputResource{}, v1.NewClientErrInvalidRequest("invalid request timeout duration")
+			}
+			var backendRequestDuration time.Duration
+			if route.TimeoutPolicy.BackendRequest != "" {
+				backendRequestDuration, err = time.ParseDuration(route.TimeoutPolicy.BackendRequest)
+				if err != nil {
+					return []rpv1.OutputResource{}, v1.NewClientErrInvalidRequest("invalid backend request timeout duration")
+				}
+			} else {
+				// If the backend request timeout is not specified, default to the request timeout
+				backendRequestDuration = requestDuration
+			}
+			// Compare the 2 request durations and ensure that the request timeout is greater than the backend request timeout
+			if requestDuration < backendRequestDuration {
+				return []rpv1.OutputResource{}, v1.NewClientErrInvalidRequest("request timeout must be greater than backend request timeout")
+			}
+			timeoutPolicy = &contourv1.TimeoutPolicy{
+				Response: route.TimeoutPolicy.Request,
```

Comment:
If we are not letting Contour know about the BackendRequest, what component knows about the BackendRequest? Do we just save it to the Database?

I guess my question is if we need BackendRequest in the first place?